src/WebJobs.Script/Description/FunctionInvokerBase.cs

```diff
@@ -63,13 +41,15 @@ internal FunctionInvokerBase(ScriptHost host, FunctionMetadata functionMetadata)
 
         public ScriptHost Host { get; }
 
+        public FunctionLogInfo LogInfo { get; }
+
         public FunctionMetadata Metadata { get; }
 
-        internal TraceWriter FileTraceWriter { get; set; }
+        protected TraceWriter TraceWriter => LogInfo.TraceWriter;
```

Comment:
I demoted the visibility it from  TraceWriter: public --> protected 



---
In reply to: [126828944](https://github.com/Azure/azure-webjobs-sdk-script/pull/1660#discussion_r126828944) [](ancestors = 126828944)